### PR TITLE
Queries 2.2

### PR DESCRIPTION
**Pull request by Evtaft22**
- refactored front end
- added middleware
- moved queries to the queries.js file
- added dummy data
- seeded dummy data into the db
- accessing user data for current session
- displaying general feed data on home page

### CODE REVIEW

**Review by bsbrionesOS — CHANGES_REQUESTED** [review]
commits failed due to travis time out, remove useeffect for toggle menu, uncomment the  s3 file

**Review by HenokSamuel — DISMISSED** [review]

File: server/s3.js [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-//     key: function (req, file, cb) {
-//       cb(null, req.s3Key);
-//     },
-//   }),
-// });
+const upload = multer({
+  storage: multerS3({
+    s3: s3,
+    bucket: AWS_BUCKET_NAME,
+    metadata: function (req, file, cb) {
```
Comment (jstrzesz) [inline server/s3.js]: change to arrow function

File: server/s3.js [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+      cb(null, { fieldName: file.fieldname });
+    },
+    key: function (req, file, cb) {
```
Comment (jstrzesz) [inline server/s3.js]: change to arrow function

**Review by HenokSamuel — APPROVED** [review]
approved functions changed

**Review by bsbrionesOS — APPROVED** [review]
requested changed approved

**Pull request merged**